modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -245,6 +247,60 @@ The other two commands are:
 
 TIP: See xref:configuration/server/background_jobs_configuration.adoc[background jobs configuration] to learn more.
 
+[[managing-background-jobs]]
+== Managing Background Jobs
+
+Use the `background:queue` command to manage background jobs.
+
+[source,console]
+----
+background:queue
+ background:queue:delete     Delete a job from the queue
+ background:queue:execute    Run a single background job from the queue
+ background:queue:status     List queue status
+----
+
+The command `background:queue:status` will list queued background jobs, including
+details when it last ran.
+
+This example lists the queue status:
+
+[source,console]
+----
+sudo -u www-data php occ background:queue:status
+
+  +----+---------------------------------------------------+---------------------------+---------------+
+  | Id | Job                                               | Last run                  | Job Arguments |
+  +----+---------------------------------------------------+---------------------------+---------------+
+  | 1  | OCA\Files\BackgroundJob\ScanFiles                 | 2018-06-13T15:15:04+00:00 |               |
+  | 2  | OCA\Files\BackgroundJob\DeleteOrphanedItems       | 2018-06-13T15:15:04+00:00 |               |
+  | 3  | OCA\Files\BackgroundJob\CleanupFileLocks          | 2018-06-13T15:15:04+00:00 |               |
+  | 4  | OCA\DAV\CardDAV\SyncJob                           | 2018-06-12T19:15:02+00:00 |               |
+  | 5  | OCA\Federation\SyncJob                            | 2018-06-12T19:15:02+00:00 |               |
+  | 6  | OCA\Files_Sharing\DeleteOrphanedSharesJob         | 2018-06-13T15:15:04+00:00 |               |
+  | 7  | OCA\Files_Sharing\ExpireSharesJob                 | 2018-06-12T19:15:02+00:00 |               |
+  | 8  | OCA\Files_Trashbin\BackgroundJob\ExpireTrash      | 2018-06-13T15:15:04+00:00 |               |
+  | 9  | OCA\Files_Versions\BackgroundJob\ExpireVersions   | 2018-06-13T15:15:04+00:00 |               |
+  | 10 | OCA\UpdateNotification\Notification\BackgroundJob | 2018-06-12T19:15:03+00:00 |               |
+  | 11 | OC\Authentication\Token\DefaultTokenCleanupJob    | 2018-06-13T15:15:04+00:00 |               |
+  +----+---------------------------------------------------+---------------------------+---------------+
+----
+
+The command `background:queue:delete` will delete a queued background job.
```

Comment:
In this sentence, please use the active tense "deletes", instead of the passive tense "will delete", because active tense is clearer and more direct.